lib/client.ts

```diff
@@ -200,6 +200,26 @@ export default class Client {
     return this.getChatMemberIds("room", roomId);
   }
 
+  private async getBotFollowers(): Promise<string[]> {
```

Comment:
You don't have to set this method to private. Just use public.

Reply:
Fixed, thanks!